src/App.jsx

```diff
@@ -14,57 +15,63 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [value, selectedGood] = useState('Jam');
```

Comment:
The `useState` hook is used correctly, but the naming of the state and the setter function is unconventional. Typically, the first element of the array returned by `useState` is the state variable, and the second is the setter function. Consider renaming `selectedGood` to `setValue` for clarity.